### ISSUE TITLE
Weird random paths are drawn after matplotlib save

### ISSUE THREAD

**Issue by grietje** [nschloe/matplotlib2tikz#78]
Consider the following MWE:

``` python
import matplotlib.pyplot as plt
import numpy as np
from matplotlib2tikz import save as tikz_save

x = np.linspace(0, 10)
plt.figure(figsize = (4, 2.5))
plt.plot(x, np.sin(x))
plt.savefig('test.pgf', bbox_inches = 'tight')
tikz_save('test.tex')
```

Inspecting the resulting file `test.tex` shows stray lines of the form

``` latex
\path [draw=black, fill opacity=0] (axis cs:1,-1)
--(axis cs:1,1);
```

When one removes the line `plt.savefig('test.pgf', bbox_inches = 'tight')` from the MWE,
then these random paths are also gone.
Maybe the check for `fill opacity=0` in https://github.com/nschloe/matplotlib2tikz/blob/master/matplotlib2tikz/path.py#L16 can be generalized?

**Comment by nschloe** [comment]
This is a really weird thing. I'd love to generalize, but I'm not sure how, to be honest. Just scratching the `fill opacity=0` condition would wipe out many legal paths.

**Comment by homocomputeris** [comment]
I have the same issue. 
In my case, it always draws lines at _x=0_, _x=1_, _y=0_ and sometimes other _y=const_.
Doesn't seem to depend on the legend, xlim, ylim, labels or the title.

**Comment by oliversheridanmethven** [comment]
I also have this issue of stray lines.

**Comment by Globulle** [comment]
Same issue for me when `plt.savefig` is called before `tikz_save`.

**Comment by TobiasWegener** [comment]
Hallo everybody,
Indeed had exactly the same problem.
I boild it down that it is caused be the line:
%pylab inline

If one exchange it with 
%matplotlib inline
from pylab import *

It seem to remove the 4 extra passes.
Hope that helps.

Thank you very much for matplotlib2tikz, it is very helpfull and much better than the default matplotlib variant of pgf plots!

**Comment by nschloe** [comment]
Finally fixed now by d3819ba340014bae6e8517e844868e5526b5bf79.
